### PR TITLE
Slvi

### PR DESCRIPTION
**Pull request by SoulVictus**

### CODE REVIEW

File: bbz-project/src/components/ContextController.js [hunk 1]
```diff
@@ -1,28 +1,39 @@
...
-    const [currentDb, setCurrentDb] = useState({});
-    const { fetchDbNames, addRecord, deleteRecord, editRecord } = useFetch("https://backend-bbz.herokuapp.com/api")
+    const [currentDbId, setCurrentDbId] = useState(0);
+    const [currentDb, setCurrentDb] = useState([]);
+    const { get, addRecord, deleteRecord, editRecord } = useFetch("https://backend-bbz.herokuapp.com/api")
+
+    useEffect(async () => {
+        const response = await get(`/request_db?db=${currentDbId}`);
+        let newRecordsList = [];
+        for(let record in response.message) {
```
Comment (xdexer) [inline bbz-project/src/components/ContextController.js:16]: Will be obsolete by Sunday, remember to delete the loop.

File: bbz-project/src/components/ContextController.js [hunk 2]
```diff
@@ -1,28 +1,39 @@
...
 const DataContext = createContext();
 
 const ContextController = (props) => {
     const [dbList, setDbList] = useState();
     const [dbNames, setDbNames] = useState();
-    const [currentDb, setCurrentDb] = useState({});
-    const { fetchDbNames, addRecord, deleteRecord, editRecord } = useFetch("https://backend-bbz.herokuapp.com/api")
+    const [currentDbId, setCurrentDbId] = useState(0);
+    const [currentDb, setCurrentDb] = useState([]);
+    const { get, addRecord, deleteRecord, editRecord } = useFetch("https://backend-bbz.herokuapp.com/api")
```
Comment (xdexer) [inline bbz-project/src/components/ContextController.js:11]: If something is a placeholder for future features please provide a comment stating that near such code.

**Review by xdexer — CHANGES_REQUESTED** [review]
Looks good to me, but before merging please provide a comment as stated in the review.

File: bbz-project/src/components/ContextController.js (hunk 1 above)
Comment (SoulVictus) [reply bbz-project/src/components/ContextController.js:16]: of course sir, can i close the ticket yet?

File: bbz-project/src/components/ContextController.js (hunk 2 above)
Comment (xdexer) [reply bbz-project/src/components/ContextController.js:12]: Looks good to me.

**Review by xdexer — APPROVED** [review]
Everything is fine, approving.

File: bbz-project/src/components/ContextController.js (hunk 2 above)
Comment (SoulVictus) [reply bbz-project/src/components/ContextController.js:12]: fixed

**Pull request merged**